angular-app/src/app/todo-shell/todo-shell.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="shell-container">
+    <app-todo-header>To Do List</app-todo-header>
+    <div class="list-container">
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+        <app-todo-item>To Do Item</app-todo-item>
+    </div>
+    <app-todo-add class="add-button"></app-todo-add>
+</div>
```

Comment:
Pentru a pastra separarea functionalitatilor, todo list trebuie trecut in modulul de home.